src/library/Server/Manager/Whm.php

```diff
@@ -98,9 +120,7 @@ public function getLoginUrl(Server_Account $account = null): string
      */
     public function getResellerLoginUrl(Server_Account $account = null): string
     {
-        $host = $this->_config['host'];
-
-        return 'http://' . $host . '/whm';
+        return $this->getLoginUrl();
     }
```

Comment:
I'm not sure this would be correct given that previously the reseller login URL was different compared to the regular URL

Reply:
oi, good eye, thats what I get for blindly following plesk.

Reply:
Actually, I think it works as long as the reseller account is passed and the request doesn't fail. According to the cPanel docs this function works for users and resellers. I may be misunderstanding how the reseller function is called though.

Reply:
Upon further review, I need to pass "whostmgrd" instead of cpaneld for resellers. So I will work on that, thank you for your insight.

Reply:
I have adjusted the code and think it SHOULD work, again unable to test myself really. New code just checks `if ($account->getReseller())` and adjusts the 'service' variable of the request.

Reply:
I decided to split the functions to reduce the chance of bugs or errors.